Code/Framework/AzFramework/CMakeLists.txt

```diff
@@ -76,6 +76,7 @@ if(PAL_TRAIT_BUILD_TESTS_SUPPORTED)
             NAMESPACE AZ
             FILES_CMAKE
                 Tests/frameworktests_files.cmake
+                ${pal_dir}/platform_${PAL_PLATFORM_NAME_LOWERCASE}_files.cmake
```

Comment:
was this missing before? I dont see these files in this review

Reply:
Yeah, this was missing previously. I had made changes to this file, but then reverted them, but it still represents files that don't get added to the resulting project (they are only trait headers).